README.md

```diff
@@ -2,19 +2,39 @@
 
 This repository compile and publish as release final binaries of the Rust toolchain targetting RISC-V RV32E.
 
-## Download 
+## Compiling
+
+```sh
+./clone.sh
+./build.sh
+./package.sh
+```
+
+Executing those steps will create a `rust-rve-nightly-$rust_commit_date-$TOOLCHAIN_HOST_TRIPLET.xz` in your working directory.
```

Comment:
should also describe how do we link this with rustup

Reply:
It's covered under `#Installation` but I added more explanation now.